dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/EnvironmentServiceImpl.java

```diff
@@ -106,8 +106,8 @@ public Map<String, Object> createEnvironment(User loginUser, String name, String
             putMsg(result, Status.DESCRIPTION_TOO_LONG_ERROR);
             return result;
         }
-        Map<String, Object> checkResult = checkParams(name, config, workerGroups);
-        if (checkResult.get(Constants.STATUS) != Status.SUCCESS) {
+        Result checkResult = checkParams(name, config, workerGroups);
+        if (checkResult.getCode() != Status.SUCCESS.getCode()) {
```

Comment:
We can directly throw ServiceException in service, use Result is not convenient.